src/components/molecules/OcTable/OcTable.vue

```diff
@@ -248,8 +255,34 @@ export default {
     },
   },
   methods: {
-    dragStart(file) {
+    dragOver(event) {
+      event.preventDefault()
+      event.target.style.cursor = "move"
```

Comment:
Local-linked it to web, there are some weird things going on regarding cursors when dropping a selection on one of the selected elements. The cursor for some elements stays in `move` mode afterwards.

Reply:
aahh damn, all because of stupid chrome who sets a different cursor when dragging :D yeah need to figure out how to reset the cursor

Reply:
ok should now work hopefully, tested it and seems to work now